lib/mi_bridges/driver/create_account_page.rb

```diff
@@ -49,19 +53,19 @@ def create_driver_application
       end
 
       def generate_user_id
-        SecureRandom.hex(10)
+        SecureRandom.hex(MAXIMUM_USER_ID_CHAR_COUNT / 2)
       end
 
       def generate_password
-        SecureRandom.hex(8)
+        SecureRandom.hex(MAXIMUM_PASSWORD_CHAR_COUNT / 2)
       end
 
       def generate_secret_question_1_answer
-        SecureRandom.hex
+        SecureRandom.hex(MAXIMUM_SECRET_QUESTION_ANSWER_CHAR_COUNT / 2)
       end
 
       def generate_secret_question_2_answer
-        SecureRandom.hex
+        SecureRandom.hex(MAXIMUM_SECRET_QUESTION_ANSWER_CHAR_COUNT / 2)
```

Comment:
Why the need to divide by 2?

Reply:
> The argument n specifies the length of the random length. The length of the result string is twice of n.

That's how secure random works - thoughts on way to clarify?

https://ruby-doc.org/stdlib-1.9.3/libdoc/securerandom/rdoc/SecureRandom.html

Reply:
So weird. The other approaches I can think of are just too verbose and wordy to really make it worthwhile.

All good in the hood.